### PR TITLE
Add default theme from Shoop repository as an addon

### PR DESCRIPTION
**Pull request by tulimaki**
Refs SHOOP-1761

### CODE REVIEW

**Comment by suutari-ai** [comment]
IMHO the theme name should be renamed to something less misleading than "Default Theme".

**Comment by suutari-ai** [comment]
How about "Basic Theme" or "Simple Theme"?

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+include README.md
```
Comment (suutari-ai) [inline MANIFEST.in]: README should be ReST. Rename it to README.rst and convert contents to ReST format. See the README.rst in Shoop for an example.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
-# shoop-default-theme
+Shoop simple theme		
+============
```
Comment (suutari-ai) [inline README.md]: Please make the length of `====` line match the length of the title.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+
+This package implements simple theme
+for the [Shoop](https://shoop.io/) platform.
+
+This theme is not actively developed.
+
+## Copyright
```
Comment (suutari-ai) [inline README.md]: Use `-------` underlining for subsection headers (as in Shoop's README.rst).

File: shoop_simple_theme/__init__.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+A Bootstrap-powered default theme for Shoop.
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]: "default theme" should be "simple theme" here.

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
-# shoop-default-theme
+Shoop simple theme
```
Comment (suutari-ai) [inline README.md]: Title case the title, i.e "Shoop Simple Theme".

File: README.md [hunk 6]
```diff
@@ -1 +1,20 @@
...
+
+Copyright (C) 2012-2015 by Shoop Ltd. <contact@shoop.io>
+
+Shoop is International Registered Trademark & Property of Shoop Ltd.,
+Business ID: FI24815722, Business Address: Aurakatu 12 B, 20100 Turku,
+Finland.
+
+## License
+
+Shoop-simple-theme is published under the GNU Affero General Public License,
```
Comment (suutari-ai) [inline README.md]: "Shoop-simple-theme" -> "Shoop Simple Theme"

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import setuptools
+
+
+if __name__ == '__main__':
+    setuptools.setup(
+        name="shoop-simple-theme",
+        version="1.0.0",
+        description="Shoop simple theme",
```
Comment (suutari-ai) [inline setup.py]: `description="Shoop Simple Theme",`

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+        packages=setuptools.find_packages(".", include=("shoop_simple_theme", "shoop_simple_theme.*",)),
```
Comment (suutari-ai) [inline setup.py]: This seems a bit too complicated. I think just
```
packages=setuptools.find_packages(),
```
should work.

File: shoop_simple_theme/__init__.py [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+"""
+from shoop.apps import AppConfig
+from shoop.xtheme.theme import Theme
+
+
+class DefaultTheme(Theme):
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]: Rename this to just Theme and import xtheme with full path, i.e.
```python
import shoop.xtheme

class Theme(shoop.xtheme.Theme):
    ...
```
That way the class name would not depend on the package name.

File: shoop_simple_theme/__init__.py [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    identifier = "shoop_simple_theme"
+    name = "Shoop Simple Theme"
+    template_dir = "default/"
+
+
+class DefaultThemeAppConfig(AppConfig):
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]: Same here:
```python
import shoop.apps

class AppConfig(shoop.apps.AppConfig):
    ...
```

File: shoop_simple_theme/__init__.py [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+    name = "shoop_simple_theme"
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]: I'd prefer
```
name = __name__
```

File: shoop_simple_theme/__init__.py [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+A Bootstrap-powered default theme for Shoop.
+"""
+from shoop.apps import AppConfig
+from shoop.xtheme.theme import Theme
+
+
+class DefaultTheme(Theme):
+    identifier = "shoop_simple_theme"
+    name = "Shoop Simple Theme"
+    template_dir = "default/"
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]: `default/` is no very good template dir name for this, maybe `shoop_simple_theme/` should be used instead?

File: shoop_simple_theme/__init__.py [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class DefaultThemeAppConfig(AppConfig):
+    name = "shoop_simple_theme"
+    verbose_name = DefaultTheme.name
+    label = "shoop_simple_theme"
+    provides = {
+        "xtheme": "shoop_simple_theme:DefaultTheme"
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]:
```
`"xtheme": __name__ + ":Theme",
```

File: shoop_simple_theme/__init__.py [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+
+default_app_config = "shoop_simple_theme.DefaultThemeAppConfig"
```
Comment (suutari-ai) [inline shoop_simple_theme/__init__.py]:
```
default_app_config = __name__ + ":AppConfig"
```

File: shoop_simple_theme/__init__.py [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class Theme(shoop.xtheme.Theme):
+    identifier = "shoop_simple_theme"
+    name = "Shoop Simple Theme"
+    template_dir = "shoop_simple_theme/"
+
+
+class AppConfig(shoop.apps.AppConfig):
+    name = __name__
```
Comment (akx) [inline shoop_simple_theme/__init__.py]: This makes it difficult to figure out what the actual name of the app is.

File: shoop_simple_theme/__init__.py (hunk 9 above)
Comment (akx) [inline shoop_simple_theme/__init__.py]: I disagree. You don't name your CBV views just `CreateView(generic.CreateView)` either.

File: shoop_simple_theme/__init__.py (hunk 10 above)
Comment (akx) [inline shoop_simple_theme/__init__.py]: I disagree (see my comment in https://github.com/shoopio/shoop/pull/252).

File: MANIFEST.in [hunk 16]
```diff
@@ -0,0 +1,4 @@
+include README.rst
+include LICENSE
+
+recursive-include shoop_simple_theme *.jinja
```
Comment (suutari-ai) [inline MANIFEST.in]: Add *.ico and *.png too.

**Pull request merged**